vars/buildPlugin.groovy

```diff
@@ -66,27 +66,20 @@ def call(Map params = [:]) {
                             isMaven = fileExists('pom.xml')
                             incrementals = fileExists('.mvn/extensions.xml') &&
                                     readFile('.mvn/extensions.xml').contains('git-changelist-maven-extension')
+                            final String gitUnavailableMessage = "[buildPlugin] Git CLI may not be available"
                             if (incrementals) { // Incrementals needs 'git status -s' to be empty at start of job
                                 if (isUnix()) {
-                                    sh(script: 'git clean -xffd > /dev/null 2>&1',
-                                       label: 'Clean for incrementals',
-                                       returnStatus: true) // Ignore failure if CLI git is not available
+                                    sh 'git clean -xffd || echo "${gitUnavailableMessage}"'
```

Comment:
Will not work: `'` will not interpolate Groovy expressions. And you should not use `"`. Just inline.